src/libraries/Microsoft.PowerFx.LanguageServerProtocol/JsonRpc/ObsoleteTypeSerializer/FormulaTypeSchema.cs

```diff
@@ -13,7 +13,8 @@ internal class FormulaTypeSchema
         [Obsolete("Use Microsoft.PowerFx.Core.SchemaTypeName instead. This JSON representation of types is not supported.")]
         public enum ParamType
         {
-            Number,
+            Number,
+            Decimal,
```

Comment:
The functions in this folder are all obsolete, and don't need to be updated.

Reply:
Still better to do the update?  Or should we remove the folder if it isn't going to be kept up to date?

Reply:
Yeah, fine to update. We'll remove them soon anyway